README.md

```diff
@@ -93,26 +93,25 @@ provisioner:
 Full configuration:
 
 ``` yml
-source: oci://ghcr.io/cirruslabs/macos-sonoma-xcode:15.3
+source: oci://ghcr.io/cirruslabs/macos-runner:sequoia
 provisioner:
   type: gitlab
   config:
     gitlabURL: <GITLAB_INSTANCE_URL>
     runnerToken: <RUNNER_TOKEN>
     executor: <EXECUTOR> # defaults to 'shell'
     maxNumberOfBuilds: <MAX_BUILDS> # defaults to '1'
-    downloadLatest: <DOWNLOAD_LATEST> # defaults to 'true'
+    downloadLatest: <DOWNLOAD_LATEST> # defaults to 'true'. If 'false' it expects the binary to be present at the user's home directory
     downloadURL: <DOWNLOAD_URL> # defaults to GitLab official S3 bucket
-    configToml: > # Advanced config as custom config.toml file to be appended to the basic config and copied to the runner.
-      <CONFIG_TOML>
+    useToml: <USE_TOML> # defaults to `false`. If `true` it ignores the other runner configurations and expects the user to have a toml file ready in the admin's home directory.

```

Comment:
The GitLab provisioner expects that the config file is called `config.toml`, right?

I think the name should also be part of the documentation. Another approach could be to have a optional string value that represents the config name instead of the `useToml` boolean.

Reply:
Good point! I have changed the implementation to use a `tomlPath` instead.